statsmodels/stats/weightstats.py

```diff
@@ -636,6 +636,89 @@ def __init__(self, d1, d2):
 #            d1.nobs1 = d1.sum_weights.astype(float)  #float just to make sure
 #        self.nobs2 = d2.sum_weights.astype(float)
 
+    @classmethod
+    def from_data(cls, data1, data2, weights1=None, weights2=None, 
+            ddof1=0, ddof2=0):
+        '''construct a CompareMeans object from data
+
+        Parameters
+        ----------
+        data1, data2 : array-like, 1-D or 2-D
+            compared datasets
+        weights1, weights2 : None or 1-D ndarray
+            weights for each observation of data1 and data2 respectively,
+            with same length as zero axis of corresponding dataset.
+        ddof1, ddof2 : int
+            default ddof1=0, ddof2=0, degrees of freedom for data1,
+            data2 respectively.
+
+        Returns
+        _______
```

Comment:
this looks like an underline instead of `----------`
most likely breaks sphinx/rst